msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd like to discuss moving to the WixAdvancedUI after we get this in. We will be able to reuse all this so we can change after this gets in. Would like to have some foundation to start tweaking/improving.